homeassistant/components/swiss_public_transport/strings.json

```
@@ -0,0 +1,27 @@
+{
+  "config": {
+    "error": {
+      "client": "Invalid request: Check at [stationboard](http://transport.opendata.ch/examples/stationboard.html) if your station names are valid",
+      "unknown": "Could not parse request response: Check at [stationboard](http://transport.opendata.ch/examples/stationboard.html) if your station names are valid"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "from": "Start Station",
```

Please use sentence case for translations

```suggestion
          "from": "Start station",
```